apps/opik-backend/src/main/java/com/comet/opik/api/resources/v1/priv/SpansResource.java

```diff
@@ -51,6 +51,8 @@
 import java.util.UUID;
 import java.util.stream.Collectors;
 
+import static com.comet.opik.api.Span.SpanPage;
+import static com.comet.opik.api.Span.View;
```

Comment:
Minor: not a big fan of static imports in general. For this particular case, you won't know which View it is.